src/pages/workspaces/workspace/Analyses.js

```diff
@@ -63,16 +62,20 @@ const AnalysisCardHeaders = ({ sort, onSort }) => {
 }
 
 const AnalysisCard = ({
-  namespace, name, lastModified, metadata, application, wsName, onRename, onCopy, onDelete, onExport, canWrite, currentUserHash,
-  potentialLockers
+  currentRuntime, namespace, name, lastModified, metadata, application, wsName, onRename, onCopy, onDelete, onExport, canWrite, currentUserHash,
+  potentialLockers, currentTool
```

Comment:
1. `currentTool` doesn't seem to be used.
2. IntelliJ Reformatter splits lines a bit differently.

```suggestion
  currentRuntime, namespace, name, lastModified, metadata, application, wsName, onRename, onCopy, onDelete, onExport, canWrite,
  currentUserHash, potentialLockers
```